blueprints/ember-visual-test/index.js

```diff
@@ -6,10 +6,24 @@ module.exports = {
   },
 
   afterInstall: function() {
-    return this.insertIntoFile('.gitignore',
-      `/visual-test-output/tmp/**/*.png
-/visual-test-output/diff/**/*.png`).then(function() {
-      return this.insertIntoFile('.npmignore', `/visual-test-output`);
-    }.bind(this))
+    return this.insertIntoFile(
+      '.gitignore',
+      '/visual-test-output/tmp/**/*.png\n/visual-test-output/diff/**/*.png'
+    ).then(() => this.insertIntoFile(
+      '.npmignore',
+      '/visual-test-output'
+    )).then(() => this.insertIntoFile(
+      'tests/test-helper.js',
+      `import setupVisualTests from 'ember-visual-test/test-support/setup';`,
```

Comment:
I think a simpler approach would be just to import a vendor file with QUnit customizations. You can see how it's done in the `ember-qunit` addon https://github.com/emberjs/ember-qunit/blob/6877684cb3b8c960ebbe06b2f58dfde0243f2b8f/index.js#L21.